source/_integrations/bthome.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: BThome Bluetooth
+description: Instructions on how to integrate BThome BLE devices into Home Assistant.
+ha_category:
+  - Sensor
+ha_bluetooth: true
+ha_release: 2022.9
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@Ernst79'
+ha_domain: bthome
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates [BThome](https://bthome.io/) BLE devices into Home Assistant.
+
+{% include integrations/config_flow.md %}
+
+The BThome BLE integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.
+
+BTHome is an energy effective but flexible BLE format to broadcast data with Bluetooth and allows you to create your own DIY BLE sensors. More information about the BThome BLE format and projects that use the format can be found on the [BThome website](https://bthome.io/).
+
+## Supported sensor measurements
+
+The current release supports the following sensor measurement types. More (binary) sensor types will be added in a future release.
```

Comment:
Let's remove this here. If we want, we can add a line below this list where people can request new sensors.

```suggestion
The current release supports the following sensor measurement types.
```

I actually don't even know if we should mention this all here, it's all in the BTHome spec?

Reply:
yes, but the BThome specs are also giving binary sensors, which are not supported in the bthome integration yet, but they are in e.g. BLE monitor. I hope to finish the other sensors before beta release, but binary sensors will follow probably in October release.

Reply:
If you leave the PR open for a few more days, I can try to add the other sensors, such that we can say that we only support sensors in the current release. We can than leave the list out. But I need two or three days to add the remaining sensors.